docs/DEVELOPING.md

```diff
@@ -39,89 +108,51 @@ source "$NVM_DIR/nvm.sh"
-1. Use ```./scripts/run_debug_webserver``` to start the mock server, if the streaming deps were not installed add the **--mock-streams** flag like ```./scripts/run_debug_webserver --mock-streams```.
-
-## Developing with a mocked out controller (with 4 stereo channel audio) on something running Raspberry Pi OS
-1. Start with a 32-bit version of Rasberry Pi OS. This needs to be older than december 2020 since our system only supports the ALSA audio backend currently.
-1. Connect a cmedia based, usb, 7.1 channel audio device to the pi. We have tested using this one: https://www.amazon.com/Vantec-NBA-200U-External-Channel-Adapter/dp/B004HXGJ3S
-1. Checkout this repo on a linux based system (a git bash shell on windows works fine as well)
-1. Edit config/asound.conf. Uncomment the "Old Prototype" section at the bottom, and comment out the similar configuration above. This should be the configuration needed for the 7.1 channel USB audio card. Depending on the setup the card will either show up as #2 or #3. That needs to be changed on lines 27 and 32. Find the card # using ```aplay -l | grep "USB Sound Device"``` and edit those lines to include the correct #.
-1. Execute ```scripts/deploy USER@HOSTNAME --mock-ctrl``` or ```scripts/deploy USER@IP_ADDRESS``` replacing USER and HOSTNAME/IP_ADDRESS with the appropriate values for the pi device
-1. Over ssh connection, run ```scripts/run_debug_webserver --mock-ctrl``` from the ```~/amplipi-dev``` directory.
-
-## Installing AmpliPi from scratch on a Pi Compute Module
-1. For a fresh pi compute module, run `scripts/bootstrap-pi`.
-   All pi compute modules shipped with AmpliPi's have already been configured using this script,
-   so this step should not be necessary.
-   Running the script without any arguments will print the instructions.
-   After this step is done SSH is enabled from a fresh Raspberry Pi OS at [amplipi.local].
-1. [amplipi.local] should now be hosted on your network.
+#### Memory Limitation for AmpliPi
+When building directly on the AmpliPi, limit memory usage during the build step:
+
+```sh
+NODE_OPTIONS=--max_old_space_size=768 npm run build
+```
+
+
+
+To develop the frontend, your strategy will bifurcate depending on your access to an Amplipi:
+
+### With an Amplipi
+You will need to edit the `scripts/run_debug_frontend` file with the IP of an Amplipi and then run said file, making sure to revert the file when you're done as to not push your local IP to the repo. This will use that Amplipi's backend so you can live develop the React app on a localhost that dynamically regenerates the app as you save your files. This is the better option, as you can have the localhost app and the `amplipi.local` app in separate tabs to see the differences between what you make and what is already there, particularly if you run a fresh `scripts/deploy` when based on the `main` branch to ensure the system is at its most up-to-date first.
```

Comment:
> You will need to edit the `scripts/run_debug_frontend` file with the IP of an Amplipi and then run said file, making sure to revert the file when you're done as to not push your local IP to the repo.

just use AMPLIPI_URL instead of this